subsys/bluetooth/host/classic/br.c

```diff
@@ -907,6 +911,42 @@ int bt_br_init(void)
 		}
 	}
 
+	buf = bt_hci_cmd_create(BT_HCI_OP_READ_DEFAULT_LINK_POLICY_SETTINGS, 0);
+	if (!buf) {
+		return -ENOBUFS;
+	}
+
+	err = bt_hci_cmd_send_sync(BT_HCI_OP_READ_DEFAULT_LINK_POLICY_SETTINGS, buf, &rsp);
+	if (err) {
+		return err;
+	}
```

Comment:
You don't need `bt_hci_cmd_create()` if you don't have any command parameters. The above is the same as:
```c
err = bt_hci_cmd_send_sync(BT_HCI_OP_READ_DEFAULT_LINK_POLICY_SETTINGS, NULL, &rsp);
```